### DISCUSSION TITLE
Playground API to get state of stack, storage and memory?

### DISCUSSION THREAD

**Discussion by hermy0** [smlxl/evm.codes#277] [General]
Is it possible to get a python api for the playground? I'd like to send opcodes and get a response with the stack, storage and memory after those opcodes.

**Comment by peroket** [comment]
Thank you for the suggestion. Unfortunately, this is not in our plans at the moment.

**Reply by hermy0** [reply]
Can I pay for a custom one?

**Reply by dorlevi** [reply]
We appreciate the offer, but we are working on other priorities now on evm.storage and our dev platform, if you want to issue a bounty/ get someone to work on this we will gladly help them with PRs and advice
